src/metrics.jl

```diff
@@ -708,20 +712,3 @@ function _pairwise!(r::AbstractMatrix, dist::CosineDist, a::AbstractMatrix)
     end
     r
 end
-
-# CorrDist
```

Comment:
Where have these gone?

Reply:
OK. Though doesn't this mean that some fallback definitions will be missing for `CorrDist`, like `parameters`?

Reply:
Since `CorrDist` doesn't call
```
evaluate(d::UnionMetrics, a::Union{Array, ArraySlice}, b::Union{Array, ArraySlice})
```
there's no need for `parameters` and `eval_*`, right?